### ISSUE TITLE
documentation page should list docs for earlier versions

### ISSUE THREAD

**Issue by mitzimorris** [stan-dev/stan-dev.github.io#103]
this page:
https://mc-stan.org/users/documentation/
should contain links to earlier versions of the docs.
online docs exist since 2.18  (functions ref added for 2.19)

**Comment by dpsimpson** [comment]
Yes. And it should clearly state which ones are relevant to RStan, PyStan,
and CmdStanX. Most users are unlikely to realize that they’re lagged.
